blobstore/backend/minio.go

```diff
@@ -0,0 +1,150 @@
+package backend
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/flynn/flynn/pkg/postgres"
+	"github.com/flynn/flynn/pkg/random"
+	"github.com/minio/minio-go"
+)
+
+func init() {
+	Backends["minio"] = NewMinio
+}
+
+func NewMinio(name string, info map[string]string) (Backend, error) {
+
+	endpoint := info["endpoint"]
+	accessKeyID := info["access_key_id"]
+	secretAccessKey := info["secret_access_key"]
+
+	useSSL := false
+	if strings.ToLower(info["use_ssl"]) == "true" {
```

Comment:
This flag should be flipped and called `insecure`, `false` by default, and the secure flag should be the opposite so that the backend uses HTTPS by default.